retroshare-gui/src/gui/notifyqt.h

```diff
@@ -127,6 +128,7 @@ class NotifyQt: public QObject, public NotifyClient
 		void transfersChanged() const ;
 		void friendsChanged() const ;
 		void lobbyListChanged() const ;
+		void ReputationChange(RsGxsId);
```

Comment:
Please, rename your signal like other, make parameter const reference, and make it const.
		void reputationChange(const RsGxsId&) const;